cost-analyzer/templates/cost-analyzer-ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ingress.enabled -}}
+
+apiVersion: v1
+data:
+  auth: YWRtaW46JGFwcjEkZ2tJenJxU2ckMWx3RUpFN1lFcTlzR0FNN1VtR1djMAo= # default is admin : admin
+kind: Secret
+metadata:
+  creationTimestamp: "2019-04-26T23:36:47Z"
+  name: kubecost-auth
+  namespace: kubecost
```

Comment:
`{{ .Release.Namespace }}` or [just omit it like the other templates](https://github.com/kubecost/cost-analyzer-helm-chart/blob/820a5031a13cd6391d3b859fade03048d28e263a/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L3-L7), right? Same with its `Ingress` friend below